### DISCUSSION TITLE
Upgrade 21.01

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1381] [Q&A] [answered]
I did an upgrade in test all went well. Applied the same in production getting spinners when trying to view a rack . Not seeing errors in the HTTP logs.  What I do see when i refresh the page I can see briefly the rack items but then straight  to spinning.  Did I miss something ?

**Comment by mfenton01** [comment]
Adding additional info Getting the following error in error_log 

[internal function]: Slim\Slim::handleErrors(2, 'strtotime(): It...', '/opt/openDCIM/o...', 68, Array)
#1 /opt/openDCIM/openDCIM-21.01/classes/Cabinet.class.php(68): strtotime('')
#2 /opt/openDCIM/openDCIM-21.01/classes/Cabinet.class.php(242): Cabinet->MakeSafe()
#3 /opt/openDCIM/openDCIM-21.01/api/v1/getRoutes.php(297): Cabinet->GetCabinet()
#4 [internal function]: {closure}('443')
#5 /opt/openDCIM/openDCIM-21.01/vendor/slim/slim/Slim/Route.php(468): call_user_func_array(Object(Closure), Array)
#6 /opt/openDCIM/openDCIM-21.01/vendor/slim/slim/Slim/Slim.php(1355): Slim\Route->dispatch()
#7 /opt/openDCIM/openDCIM-21.01/vendor/slim/slim/Slim/Middleware/Flash.php(85): Slim\Slim->call()
#8 /opt/openDCIM/openDCIM-21.01/vendor/slim/slim/Slim/Middleware/MethodOverride.php(92): Slim\Middleware\Flash->call()
#9 /opt/openDCIM/openDCIM-21.01/vendor/slim/slim/Slim/Middleware/PrettyExceptions.php(67): Slim\Middleware\MethodOverride->call()
#10 /opt/openDCIM/openDCIM-21.01/vendor/slim/slim/Slim/Slim.php(1300): Slim\Middleware\PrettyExceptions->call()
#11 /opt/openDCIM/openDCIM-21.01/api/v1/index.php(193): Slim\Slim->run()

**Comment by wilpig** [comment, marked as answer]
I'm going to guess that you don't have a timezone set in the php.ini but I thought we finally fixed that error

https://github.com/opendcim/openDCIM/blame/master/facilities.inc.php#L31

**Comment by mfenton01** [comment]
It's not .. but my php.ini  running in test  is the same in production ..
not getting errors there ..  could be something else

/pictures , /drawings files are there with write access

**Comment by mfenton01** [comment]
Just FYI editing the php.ini did fix it
